phil/s013-course-periods.md

```diff
@@ -81,6 +81,19 @@ in the export for BE
 ]
 ```
 
+# Learning Guide
```

Comment:
I think that since the JSON format changes between a student and teacher these should be different endpoints or have a `./guide/periods/1` endpoint. If we do this, should we make the perf report (and roster) in the same way?